tests/integration/targets/ecs_cluster/tasks/full_test.yml

```diff
@@ -846,6 +730,13 @@
       debug:
         msg: "***** TESTING COMPLETE. COMMENCE TEARDOWN *****"
 
+    - name: remove setup ec2 instance
+      ec2_instance:
+        instance_ids: '{{ setup_instance.instance_ids }}'
+        state: absent
+        wait: yes
```

Comment:
I would imagine we probably don't need to wait here, but I could be wrong!

Reply:
Yes. That's it. At least, the VPC must be removed at the end, because the number of vpc is limited on every account and might block all other running integration tests then.